corehq/ex-submodules/pillowtop/checkpoints/manager.py

```diff
@@ -35,7 +35,7 @@ def reset_checkpoint(checkpoint_id):
 class PillowCheckpoint(object):
 
     def __init__(self, checkpoint_id):
-        self.checkpoint_id = checkpoint_id
+        self.checkpoint_id = 'temp_{}'.format(checkpoint_id)
```

Comment:
this ensures we don't overwrite the current checkpoint. that makes sense to me. looks good